AutomatticTracks/build.gradle

```diff
@@ -11,32 +11,31 @@ repositories {
 
 dependencies {
     implementation platform('io.sentry:sentry-bom:6.3.0')
-    implementation "io.sentry:sentry-android"
-    implementation "io.sentry:sentry-android-okhttp"
-    implementation "io.sentry:sentry-android-fragment"
+    implementation 'io.sentry:sentry-android'
+    implementation 'io.sentry:sentry-android-okhttp'
+    implementation 'io.sentry:sentry-android-fragment'
```

Comment:
Suggestion (💡: Btw, a quick FYI for your @wzieba . I am always in favor of having double quotes (`"`) instead of single quotes (`'`) like that.

This is twofold and because:
1. You get a warning/error (a 🔴 underline at least) like that if you try to use a syntax like `'androidx.annotation:annotation:${version}'`, not the `${version}`, see here: `It looks like you are trying to substitute a version variable, but using single quotes ('). For Groovy string interpolation you must use double quotes (").` 
2. If we were to ever migration to Kotlin Gradle DSL, this would become a prerequisite, to have double quotes (`"`) everywhere.

Reply:
Thanks, moved all of them to `"`!